src/test/kotlin/no/nav/meldeplikt/meldekortservice/database/TestQueries.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.meldeplikt.meldekortservice.database
+
+import java.sql.Connection
+
+fun Connection.slettAlleInnsendteMeldekort() =
+    prepareStatement("""DELETE FROM INNSENDT_MELDEKORT""")
```

Comment:
Ikke så viktig her, men normalt for å tømme en tabell brukes "truncate table tabellnavn"